components/booking/src/init.spec.js

```diff
@@ -258,61 +258,61 @@ describe("scheduler component", () => {
     });
     // TODO: test time/reorder deduplication (same time, different ordering)
     // TODO: test slot-click, emitted-event
-    // TODO: test slot de-select, scheduler back to options list
+    // TODO: test slot de-select, booking back to options list
   });
 
   describe("Inherits and passes properties", () => {
     it("has a default event title", () => {
-      schedulerTestComponent.slots_to_book = basic_slots_to_book;
-      cy.get(schedulerTestComponent).find("h3").contains("Meeting:");
+      bookingTestComponent.slots_to_book = basic_slots_to_book;
+      cy.get(bookingTestComponent).find("h3").contains("Meeting:");
     });
     // TODO - Re-enable this once we have dedicated test components
     xit("inherits event title from editor-manifest", () => {
       cy.document().then(($document) => {
-        $document.getElementsByTagName("nylas-scheduler")[0].remove();
-        let newScheduler = $document.createElement("nylas-scheduler");
-        newScheduler.setAttribute("id", "test-scheduler");
-        newScheduler.slots_to_book = basic_slots_to_book;
-        $document.body.getElementsByTagName("main")[0].append(newScheduler);
+        $document.getElementsByTagName("nylas-booking")[0].remove();
+        let newBooker = $document.createElement("nylas-booking");
+        newBooker.setAttribute("id", "test-scheduler");
+        newBooker.slots_to_book = basic_slots_to_book;
+        $document.body.getElementsByTagName("main")[0].append(newBooker);
       });
       cy.get("h3").contains("My Wonderful Event:");
     });
     it("inherits event title from passed-property if no event title", () => {
       cy.document().then(($document) => {
-        $document.getElementsByTagName("nylas-scheduler")[0].remove();
-        let newScheduler = $document.createElement("nylas-scheduler");
-        newScheduler.setAttribute("id", "test-scheduler");
-        newScheduler.editor_id = "test-schedule-editor";
-        newScheduler.event_title = "Test-Passed Title";
-        newScheduler.slots_to_book = basic_slots_to_book;
-        $document.body.getElementsByTagName("main")[0].append(newScheduler);
+        $document.getElementsByTagName("nylas-booking")[0].remove();
+        let newBooker = $document.createElement("nylas-booking");
+        newBooker.setAttribute("id", "test-scheduler");
```

Comment:
```suggestion
        newBooker.setAttribute("id", "test-booking");
```

Reply:
see above re: component ID and manifest/database